math/vecops/test/vecops_rvec.cxx

```diff
@@ -106,7 +106,6 @@ TEST(VecOps, MoveCtor)
 {
    ROOT::VecOps::RVec<int> v1{1, 2, 3};
    ROOT::VecOps::RVec<int> v2(std::move(v1));
-   EXPECT_EQ(v1.size(), 0u);
```

Comment:
For `std::vector`, the standard guarantees that `v1.empty()` returns `true` for moved-from objects. I'm not sure about `size`, but couldn't `v2`'s move-constructor set it to zero?

Reply:
If the standard "guarantees that v1.empty() returns true for moved-from objects." the it also (implicitly or explicitly) must guarantee that v1.size is zero.